### ISSUE TITLE
UnhandledPromiseRejectionWarning with rollup plugin

### ISSUE THREAD

**Issue by baerkins** [ko-yelie/gulp-rollup-each#11]
I am getting the following error: 

```
(node:93222) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): [object Object]
(node:93222) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

I have narrowed it down to including the [`rollup-plugin-commonjs`](https://www.npmjs.com/package/rollup-plugin-commonjs) plugin. If I uncomment that plugin, works 👍, but unfortunately, I need `rollup-plugin-commonjs` for a few npm packages in my project.

If I use a regular rollup config with the same files and config settings, instead of `gulp-rollup-each`, the `rollup-plugin-commonjs` plugin works perfectly fine.

If you have any thoughts on this issue, or need any further details, let me know!

**Comment by ko-yelie** [comment]
@baerkins Thank you for the information.
Could you provide a sample of your gulpfile?

**Comment by baerkins** [comment]
Sure thing!

**gulpfile.js:**

```javascript
const rollup = require('rollup');
const rollupEach = require('gulp-rollup-each');
const rollupBabel = require('rollup-plugin-babel');
const rollupResolve = require('rollup-plugin-node-resolve');
const rollupCommon = require('rollup-plugin-commonjs');
const rollupESLint = require('rollup-plugin-eslint');
const rollupUglify = require('rollup-plugin-uglify');

const yargs = require('yargs').argv;
const sourcemaps = require('gulp-sourcemaps');

// Definitions
const config = {
  production: yargs.production ? yargs.production : false,
  js: {
    dest: './dist/assets/js/',
    echo: 'src/assets/echo/js/echo.js',
  }
};

// Rollup plugins
const rollupPlugins = [
  rollupBabel(),
  rollupResolve({
    jsnext: true,
    main: true
  }),
  rollupCommon(),		// If you comment this out, everything works as expected
  rollupESLint()
];

// Rollup main js files
// --------------------
gulp.task('echo:js', () => {
  return gulp.src([
    'src/assets/echo/js/*.js',
    '!src/modules/*.js'
  ])
    .pipe(gulpIf(!config.production, sourcemaps.init()))
      .pipe(rollupEach({
          plugins: rollupPlugins
        }, {
          format: 'iife'
        })
      )
    .pipe(gulpIf(!config.production, sourcemaps.write('.')))
    .pipe(gulp.dest('./dist/assets/echo/js/'));
});
```

**package.json** (for version numbers, if helpful)
```json
{
  "name": "echo",
  "version": "1.0.0",
  "main": "index.js",
  "devDependencies": {
    "autoprefixer": "^8.5.0",
    "babel-core": "^6.26.3",
    "babel-plugin-external-helpers": "^6.22.0",
    "babel-preset-env": "^1.7.0",
    "beautify": "0.0.8",
    "browser-sync": "^2.24.4",
    "cssnano": "^3.10.0",
    "del": "^3.0.0",
    "echo-engine": "^1.0.0",
    "eslint-config-prettier": "^2.9.0",
    "fancy-log": "^1.3.2",
    "gulp": "^3.9.1",
    "gulp-file": "^0.4.0",
    "gulp-flatmap": "^1.0.2",
    "gulp-gh-pages": "^0.5.4",
    "gulp-if": "^2.0.2",
    "gulp-plumber": "^1.2.0",
    "gulp-postcss": "^7.0.1",
    "gulp-rename": "^1.2.3",
    "gulp-rollup-each": "^2.1.0",
    "gulp-sass": "^4.0.1",
    "gulp-sourcemaps": "^2.6.4",
    "gulp-uglify": "^3.0.0",
    "prismjs": "^1.15.0",
    "rollup": "^0.58.2",
    "rollup-plugin-babel": "^3.0.4",
    "rollup-plugin-commonjs": "^9.1.3",
    "rollup-plugin-eslint": "^4.0.0",
    "rollup-plugin-json": "^3.0.0",
    "rollup-plugin-node-resolve": "^3.3.0",
    "rollup-plugin-uglify": "^4.0.0",
    "rollup-stream": "^1.24.1",
    "run-sequence": "^2.2.1",
    "vinyl-buffer": "^1.0.1",
    "vinyl-source-stream": "^2.0.0",
    "yargs": "^11.1.0"
  },
  "dependencies": {}
}
```

**Comment by ko-yelie** [comment]
@baerkins The reason is that the Rollup version on which `gulp-rollup-each` depends is old. (ref https://github.com/rollup/rollup-plugin-commonjs/issues/325)
I updated the version of Rollup that `gulp-rollup-each` depends on, and released it in [v2.1.1](https://github.com/ko-yelie/gulp-rollup-each/releases/tag/v2.1.1).
Please update the version of `gulp-rollup-each` to v2.1.1.

If you have the same problem in the future, please try [the `rollup` option](https://github.com/ko-yelie/gulp-rollup-each#rollup).
```js
      .pipe(rollupEach({
          plugins: rollupPlugins
        }, {
          format: 'iife'
        }, rollup)
      )
```

**Comment by baerkins** [comment]
Apologies for never getting back - thanks so much for this fix!!
